operations/helm/charts/mimir-distributed/templates/federation-frontend/federation-frontend-dep.yaml

```diff
@@ -62,6 +62,8 @@ spec:
               mountPath: /license
             - name: active-queries
               mountPath: /active-query-tracker
+            - name: runtime-config
```

Comment:
this isn't rendered when the FF is deployed on its own 

https://github.com/grafana/mimir/blob/e32b5e2531d40677f9793f8c72ed18aa79fcf50b/operations/helm/charts/mimir-distributed/templates/runtime-configmap.yaml#L1-L0

i think this change will break the federation-frontend. It also wasn't mentioned in the PR description, so i'm a bit puzzled